include/arc_utilities/eigen_std_conversions.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef LGV_EIGEN_STD_CONVERSIONS
+#define LGV_EIGEN_STD_CONVERSIONS
+
+#include <Eigen/Dense>
+#include <vector>
+#include <cstdlib>
+
+template<typename Output>
+Output ConvertTo(Eigen::VectorXd const& vec);
+
+template<>
+std::vector<double> ConvertTo<std::vector<double>>(Eigen::VectorXd const& vec)
+{
+    std::vector<double> std_vec(vec.size());
```

Comment:
one-liner
`return std::vector<double>(vec.data(), vec.data() + vec.size() * sizeof(double));`

Reply:
nice. and turns out both methods have implicit sign conversion so I've added a static_cast to make it clear to the reader.